hivemq-edge/src/frontend/src/components/rjsf/BatchSubscription/BatchUploadButton.tsx

```diff
@@ -42,11 +42,16 @@ const BatchUploadButton: FC<BatchUploadButtonProps> = () => {
         scrollBehavior="inside"
       >
         <ModalOverlay />
-        <ModalContent>
+        <ModalContent
+          style={
+            // TODO[modal-portal] Feels like a hack. Check for side-effects
+            { position: 'static' }
```

Comment:
question(non-blocking): I do not know what this line does but it feel certainly like a hack. I you like we can look into that one together. But if it does the trick it fine to me.

Reply:
The `Select` is mangled if it drops within the container of the modal.

 ![Screenshot 2024-05-30 at 10 22 17](https://github.com/hivemq/hivemq-edge/assets/2743481/3c6b9ded-5aad-4dd0-8298-cd2942da2b47)

There is a workaround, using the `menuPortalTarget` to point to the modal element. 
However, due to the modal being positioned `relative`, the dropdown is shifted around. 

![Screenshot 2024-05-30 at 10 22 34](https://github.com/hivemq/hivemq-edge/assets/2743481/9b869920-97f1-4f3b-bea4-85eab7e7da12)


`static` restores the positioning, while no visible effect on the modal (inherited property I would guess)

![Screenshot 2024-05-30 at 10 22 49](https://github.com/hivemq/hivemq-edge/assets/2743481/3e83d2e6-2de8-4f7f-ade5-cf7fc0c03b0c)